src/Relude_Set.re

```diff
@@ -39,43 +293,44 @@ module type SET = {
   let split: (value, t) => ((t, t), bool);
 };
 
-module WithOrd = (M: BsAbstract.Interface.ORD) : (SET with type value = M.t) => {
+module WithOrd = (M: BsAbstract.Interface.ORD)
+// : (SET with type value = M.t)
```

Comment:
Oh yes, I was simply testing the effect of type annotations on the JS output, out of curiosity. I'll go ahead and uncomment that.

Reply:
Also, I don't have any reason to keep those module types around. But they were definitely useful while refactoring these modules, since they provided compiler feedback if I missed anything, e.g. if I left out a function that was previously included.

Reply:
Updated.